src/net_processing.h

```diff
@@ -90,4 +90,7 @@ struct CNodeStateStats {
 /** Get statistics from node state */
 bool GetNodeStateStats(NodeId nodeid, CNodeStateStats &stats);
 
+/** Relay transaction to every node */
+void RelayTransaction(const CTransaction& tx, CConnman* connman);
```

Comment:
style-nit: connman can be a const reference instead of a mutable raw pointer, since we don't modify connman, but push the tx to the set of all nodes.

```suggestion
void RelayTransaction(const CTransaction& tx, const CConnman& connman);
```

Reply:
style-nit: (feel free to ignore)

It is sufficient to pass the txid (as we assume the tx is in the mempool). So maybe make the signature similar to `ChainImpl`'s `void relayTransaction(const uint256& txid)` and use it there as well?